packages/UI/src/components/organisms/cards/PositionCard/index.ts

```diff
@@ -8,9 +8,9 @@ export interface PositionCardProps {
   content: {
     imgUrl: string;
     expectations: string[];
-    timeCommittment: string;
-    role: string;
-    tags: string[];
+    commitmentHoursPerWeek: string;
+    description: string;
+    skills: string[];
```

Comment:
why changing those? changing those can result in broken components that use the old props, please revert this change

Reply:
1: commitmentHoursPerWeek: strind; This is because it is what is present in the API and timeCommittment: string is not
2: description: string; This is because it is what is present in the API and role: string is not
3:     skills: string[]; This is because it is what is present in the API and tags: string[]; is not. Furthermore,  Kris let me know tags: srting[] was changed to skills: string[]

The API I refer to is https://api-staging.devlaunchers.org/documentation#/Opportunities

But I will revert it back to test for the changes again, is it okay?

Reply:
After you have considered it and checked if there is still a need to change all that I will change it right away